strapi/strapi
# Fix typescript plugin generation

### What does it do?

- update typescript plugin generation files by correcting the Strapi interface import
- removed `@strapi/strapi`  package from plugins devDependency

### Why is it needed?

- Typescript plugins are failing to run correctly 

### How to test it?

- generate a typescript plugin using `strapi generate plugin`
- navigate to the plugin and run `yarn` and `yarn develop`

The plugin should compile and work just fine

### Related issue(s)/PR(s)

fixes #13998
